sound/hda/hdac_controller.c

```diff
@@ -135,6 +136,88 @@ static unsigned int azx_command_addr(u32 cmd)
 	return addr;
 }
 
+/* receive a response via Immediate Response */
+static int snd_hdac_bus_wait_for_pio_response(struct hdac_bus *bus,
```

Comment:
@ujfalusi doesnt it make sense to only add the bool in this patch and add the helpers for PIO helpers in the next patch where they're actually used?